### DISCUSSION TITLE
removing only EL  e.g.  BL+EL+RPU => BL+RPU

### DISCUSSION THREAD

**Discussion by malibbs** [quietvoid/dovi_tool#323] [Q&A] [answered]
Hi,
i found solution for removing EL+RPU
"remove
Removes the enhancement layer and RPU data from the video."

but not for only removing "enhancement layer"

any idea?
thanks a lot

**Comment by quietvoid** [comment, marked as answer]
`convert --discard` should do what you want.

**Reply by malibbs** [reply]
Worked, although MediaInfo still shows BL+**EL**+RPU !?

**Reply by quietvoid** [reply]
It is still profile 7.

**Reply by malibbs** [reply]
o.k., I understand  :-)

Is there a command to show the Profile or Info for a h.265 video ?
Command  "info -s" results in: 
_Parsing RPU file...
Error: Found invalid RPU: Index 0, error: Invalid RPU length: 7_

**Reply by quietvoid** [reply]
No.
